### PR TITLE
Mes 6242 standard test slot template

### PR DESCRIPTION
**Pull request by piercemorris**
## Description

 - Lift and shift the test slot component
 - Connect the component to the journal store
 - Update styling, colours and alignment
 - Update icons
 - Comment out code to be implemented at a later date

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### Local

Screenshot 2021-02-02 at 10 33 41

### iOS

Simulator Screen Shot - iPad Pro (10 5-inch) updated - 2021-02-02 at 10 32 15

### CODE REVIEW

File: src/app/pages/journal/journal.page.ts [hunk 1]
```diff
@@ -153,7 +147,7 @@ export class JournalPage extends BasePageComponent implements OnInit {
     this.merged$ = merge(
       selectedDate$.pipe(map(this.setSelectedDate)),
       // completedTests$.pipe(map(this.setCompletedTests)),
-      // slots$.pipe(map(this.createSlots)),
+      slots$.pipe(map(this.createSlots)),
       slots$.pipe(map((value) => console.log(JSON.stringify(value)))),
```
Comment (tomcrawleyy) [inline src/app/pages/journal/journal.page.ts]: this line with the console log ought to be removed here now

File: src/app/providers/slot-selector/slot-selector.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        }
+      }
+    });
+    return gotValue;
+  };
+
+  public isTestSlot = (slot: Slot) => has(slot, 'vehicleTypeCode');
+
+  private resolveComponentName = (slot: SlotItem) => {
+    console.log(slot);
```
Comment (tomcrawleyy) [inline src/app/providers/slot-selector/slot-selector.ts]: console.log, to be removed?

File: src/components/test-slot/language/__tests__/language.spec.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+import { By } from '@angular/platform-browser';
+import { configureTestSuite } from 'ng-bullet';
+import { LanguageComponent } from '../language';
+
+xdescribe('LanguageComponent', () => {
```
Comment (tomcrawleyy) [inline src/components/test-slot/language/__tests__/language.spec.ts]: all of these tests are excluded, were they failing?

File: src/components/test-slot/location/__tests__/location.spec.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { configureTestSuite } from 'ng-bullet';
+import { LocationComponent } from '../location';
+
+xdescribe('LocationComponent', () => {
```
Comment (tomcrawleyy) [inline src/components/test-slot/location/__tests__/location.spec.ts]: same here

**Review by szabikr — APPROVED** [review]
Looks good

File: src/components/test-slot/location/__tests__/location.spec.ts (hunk 4 above)
Comment (piercemorris) [reply src/components/test-slot/location/__tests__/location.spec.ts]: They were failing initially but they work as expected, thanks for spotting it

**Review by tomcrawleyy — APPROVED** [review]
great work!

**Pull request merged**